bundles/org.openhab.binding.pushbullet/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -16,19 +16,19 @@
 
 		<config-description>
 			<parameter name="name" type="text" required="false">
-				<label>Name</label>
-				<description>Explicit Name of Bot, if wanted</description>
+				<label>@text/thing-type.config.pushbullet.bot.name.label</label>
+				<description>@text/thing-type.config.pushbullet.bot.name.description</description>
```

Comment:
Help me out here, what is the purpose of replacing the default labels and descriptions with these i18n placeholders?

Reply:
I am mirrored what was done previously with the other i18n placeholders for this binding such as the `addon.xml` or addon resources.

I don't usually do this but I wanted to keep the consistency. I can revert that change if necessary.

Reply:
Yes please revert.

Reply:
Should I update `addon.xml` as well?

Reply:
I ended updating that file as well for consistency.